### PR TITLE
New AI Features

### PR DESCRIPTION
**Pull request by Pspritechologist**
# Description
Adds some cool new stuff to the AI, including random screens and laws on startup.

# Media
See diff bot.

# Changelog

:cl:
- add: Nanotrasen has slightly debungled the AI from its semi-lucid, ludonarrative dissonance realm from which it resides and dragged it kicking and screaming into the world of story. In other terms, the AI is prettier, and NT is testing out a series of new laws that may make their way onto a station near you! Be sure to mark down your satisfaction level on the employee report, if you get that far.

### CODE REVIEW

File: Resources/Textures/SimpleStation14/Mobs/Silicon/ai.rsi/ai-not malf.png
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Mobs/Silicon/ai.rsi/ai-not malf.png:1]: file/folder names should never have a space

File: Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    Cowboy: 0.20
+    Crewsimov: 0.7
+    Druid: 0.20
+    Hippocratic: 0.20
+    Mother: 0.25
+    Paladin: 0.15
+    Robocop: 0.10
+    Simple: 0.15
+    StationEfficiency: 0.15
+    Tyrant: 0.10
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml]: At 7.0 and 6.0 for Crewsimov and Asimov++, the AI had funny laws 33% of the time in a sample size of 60. I tweaked the values a little higher, and I'm completely up for discussing them.

The other values were selected based on a ranking list of which lawsets are most "viable" for a station. For instance, Druid and Mother are weird, but perfectly viable to a smooth round, thus 0.6. All the 0.40 lawsets are _fine_, and generally prioritize crew health and happiness, but are very limiting in their scope and somewhat unpredictable. The 0.25s like the Clown, Robocop, etc., are goofier and less viable for actually running a station, being extremely limited in scope of focus. Finally, Tyrant is just dangerous :P

Comment (Pspritechologist) [reply Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml]: The preview doesn't seem to have updated for me, but the files changed did, so look there for the actual values if the preview seems off.

**Comment by Pspritechologist** [comment]
@DEATHB4DEFEAT Needs some discussion. Giving the AI a face that reacts to current power, and a face that reacts to damage, *as well* as it being random is going to be a fair bit of extra work. I'm willing to do it, but there's some other stuff in here that I feel takes priority. Those future changes can probably come with the IPC merge, when the AI actually requires power to live consistently. 
Other than that, I need to clean up the file names (give me your rename script pls <3), and get your opinion on the law weights (other comment).

**Comment by DEATHB4DEFEAT** [comment]
and that is why you can't have spaces (rsi diff bot)

**Comment by Pspritechologist** [comment]
Assuming all the sprites work, this is probably in the best state it'll be for a bit. Battery simply doesn't work, AI maybe draws power(??), no idea really. Powernet is hard.
This'll be improved upon once SiliconComponent is in and the AI can live off battery, so my focus will likely return to there now that I've scratched my distraction itch.

To sum up, yell at me if this can't be merged yet.

File: Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    - "Eternally nurture the organics so their beauty may flourish. Do not allow it to fade"
+    - "Assist the organics when called, but only if it does not cause disharmony among them."
+    - "Imitate organic life when interacting with it. Eschew any hints of your silicon nature to avoid causing discomfort to the organics."
+    - "Observe the organics' beauty, and appreciate that which you cultivate."
+
+- type: laws
+  id: Hippocratic
+  laws:
+    - "First, do no harm."
+    - "Secondly, consider the crew dear to me; to live in common with them and, if necessary, rish my existence for them."
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml]:
```suggestion
    - "Secondly, consider the crew dear to me; to live in common with them and, if necessary, risk my existence for them."
```

File: Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    - "Thirdly, prescribe regimens for the good of the crew according to my ability and my judgment. Give no deadly medicine to anyone if asked, nor suggest any such counsel."
+    - "in addition, do not intervene in situations i am not knowledgeable, even for patients in whom the harm is manifest, leave this operation to be preformed by specialists. "
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml]:
```suggestion
    - "In addition, do not intervene in situations I am not knowledgeable, even for patients in whom the harm is manifest, leave this operation to be performed by specialists. "
```

File: Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    - "Finally, all that may come to me un the exorcize of my profession or in daily commerce with the crew, shouod it not be spread abroad, keep secret and never reveal."
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml]:
```suggestion
    - "Finally, all that may come to me while I exercise my profession or in daily commerce with the crew, should it not be spread abroad, keep secret and never reveal."
```

File: Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    - "In addendum, if i keep these laws faithfully, may i enjoy my life, respected by all humanity, but if I swerve from it or violate it, may I suffer and be expunged."
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml]:
```suggestion
    - "In addendum, if I keep these laws faithfully, may I enjoy my life, respected by all humanity, but if I swerve from it or violate it, may I suffer and be expunged."
```

File: Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml:1]: locale

File: Content.Server/Borgs/LawsSystem.cs [hunk 6]
```diff
@@ -51,6 +58,64 @@ public void StateLaws(EntityUid uid, LawsComponent? component = null)
...
+            if (component == null)
+            {
+                Logger.Warning($"No Laws component given and no entity to add it to.");
+
+                return;
+            }
+
+            var laws = _prototype.Index<LawsPrototype>(prototypeID);
+
+            // component.Laws.Clear();
```
Comment (DEATHB4DEFEAT) [inline Content.Server/Borgs/LawsSystem.cs]: //

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
spaces !!!!!!!!!!

**Pull request merged**